contracts/sol6/Dao/KyberDAO.sol

```diff
@@ -57,7 +57,7 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5, DaoOperator
 
     // minimum duration in seconds for a campaign
     uint256 public minCampaignDurationInSeconds = 345600; // around 4 days
-    IERC20 public kncToken;
+    IERC20 public override kncToken;
```

Comment:
why this becomes override.
the function was added in IKyberStaking

Reply:
Because the IKyberDAO interface has the public getter method `kncToken`, so that when setting the DAO contract in Staking, can verify the token address

Reply:
Yeah it is how override in sol6 works :'(